.github/actions/log-metric/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Log Metric'
+description: 'Log data point to a metric with the provided value. If the metric is not there, it will create one.'
+inputs:
+  metric-name:
+    description: 'target metric name'
+    required: true
+  value:
+    description: 'the value to log to the metric'
+    required: true
+  AWS_ACCESS_KEY_ID:
+    description: 'target AWS account credentials'
+    required: true
+  AWS_SECRET_ACCESS_KEY:
```

Comment:
This is the first time we add secrets to composite actions; do you know what the recommended ways to pass secrets are?

Reply:
I'm looking up too! Looks like `configure-aws-credentials` use `inputs` to use secrets too: https://github.com/aws-actions/configure-aws-credentials/blob/8a84b07f2009032ade05a88a28750d733cc30db1/action.yml#L7-L18

Reply:
Good catch. VS Code linter didn't allow me to add `env`, but I found on the internet that people just did it anyway and it worked, so let me try it out.

Reply:
Looking at the log, it scraps off secrets plaintext too: https://github.com/aws-amplify/amplify-ui/runs/7436708485?check_suite_focus=true

Reply:
Thanks for looking! I imagine what you have is safe too, I'm just trying to find an official source to solidify this.

Reply:
New commit pushed

Reply:
Resolving as per MJ's research